recipes/mo-catnip/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "mo-catnip" %}
+{% set version = "1.0.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 65d1c726662069114d4bba6226d621fc0a8d1250b2210ed23763573ab8e87d8c
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
```

Comment:
Dost this install all the required packages from pip?

Reply:
No, all the required packages needed for proper importing need to be listed under the `run` dependencies below and need to be available from conda-forge already (or added in this PR).